azure/templates/run-tests.yml

```diff
@@ -56,12 +71,17 @@ steps:
                 testResultsFiles: "$(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/smoketest-report.xml"
                 failTaskOnFailedTests: true
     - ${{ if not(parameters.full) }}:
+          - template: "azure/components/get-aws-secrets-and-ssm-params.yml@common"
```

Comment:
Can remove this change, you've already loaded the secret between lines 21 and 24.